sentinel/src/schedule/outbound.js

```diff
@@ -159,6 +159,20 @@ const send = (payload, config) => {
         });
     }
 
+    if (authConf.type.toLowerCase() === 'header') {
+      if (authConf.name.toLowerCase() === 'authorization') {
```

Comment:
If they don't provide an `authConf.name` this line will uncleanly crash the parsing.

Note on line 147 (and 143 before it) we deal with this for properties that are relevant generically. For you here, you can probably just change it to:

```suggestion
      if (authConf.name && authConf.name.toLowerCase() === 'authorization') {
```

Reply:
Nit: I don't think there is that much value in having two if blocks nested inside one another here, you can just check both at the same time and save some precious whitespace.

Reply:
Thanks, the intention was to keep clearly distinct blocks for each authConf type.